app/lib/notifications.js

```diff
@@ -5,6 +5,9 @@ const slackLib = require('./slack');
 const ACTIVITY_ALL = 'all';
 
 module.exports = (Sequelize, activity) => {
+  if(typeof activity.GroupId === 'undefined' || typeof activity.type === 'undefined') {
```

Comment:
Indeed I didn't properly test this, was using an `activity` data structure from one of the unit-tests. I tested with live data and PR is here: https://github.com/OpenCollective/api/pull/307